src/command_modules/azure-cli-resource/HISTORY.rst

```diff
@@ -5,6 +5,7 @@ Release History
 2.1.3
 +++++
 * `provider operation show`: exception handling to exit with code 3 upon a missing resource for consistency.
+* `managedapp create --kind MarketPlace`: fixed bug in create_application where positional arguments were used instead of key-word arguments when instantiating a Plan Model.
```

Comment:
Change the history so it only reflects what is user-facing.

Reply:
"Fixed issue where users were unable to submit parameters in JSON format when creating certain kinds of managed apps."

Reply:
Okay I will make this update. I confirmed with Shilpa and the issue she had wasn't actually because of the json parameters.